examples/typed-derive.rs

```diff
@@ -19,7 +19,7 @@ struct Args {
 
     /// Allow human-readable durations
     #[arg(long)]
-    sleep: Option<humantime::Duration>,
+    sleep: Option<jiff::SignedDuration>,
```

Comment:
I've not checked, does this get the human-format parsing or does that require further opt-in?